crates/astria-core/src/primitive/v1/asset/denom.rs

```diff
@@ -75,6 +75,15 @@ impl Denom {
         };
         trace
     }
+
+    /// Calculates the length of the display formatted [Denom] without allocating a String.
+    #[must_use]
+    pub fn display_len(&self) -> usize {
+        match self {
+            Denom::TracePrefixed(trace) => trace.display_len(),
+            Denom::IbcPrefixed(_) => 68, // "ibc/" + 64 hex characters
```

Comment:
impl this fixed number on the ibc type as well and delegate there, for consistency.